src/resources/Activities/tests/Activities.spec.ts

```diff
@@ -44,7 +44,7 @@ describe('Activity', () => {
 
             activity.list(params, activityFacet);
             expect(api.post).toHaveBeenCalledTimes(1);
-            expect(api.post).toHaveBeenCalledWith(`${Activity.getBaseUrl()}/public`, activityFacet);
+            expect(api.post).toHaveBeenCalledWith(`${Activity.getBaseUrl()}`, activityFacet);
```

Comment:
Could you add tests when it's public?

Reply:
I reached out to #content-ingestion team to make the route public so that we can just use the /activty route and not /activity/public one.  I am waiting to hear back from them before adding tests that are likely going to be repetitive and deprecated.

Reply:
added the unit tests!